reference-implementation/lib/writable-stream.js

```diff
@@ -130,27 +136,17 @@ function WritableStreamAbort(stream, reason) {
   if (state === 'errored') {
     return Promise.reject(stream._storedError);
   }
+  if (stream._pendingAbortRequest !== undefined) {
+    return Promise.reject(new TypeError('Aborting in progress'));
+  }
 
   assert(state === 'writable', 'state must be writable');
 
   const error = new TypeError('Requested to abort');
-  if (stream._pendingAbortRequest !== undefined) {
+  if (stream._pendingAbortRequest === true) {
```

Comment:
It was a no-op. There was a correct check above. I have removed the redundant check.

Coverage says the real check was being tested, but I can't find the test that does it. Maybe it is a piping one? Anyway, I added an explicit test for double aborting just to make it clear.